### ISSUE TITLE
Section 7.1 refers to nonexistent `item` property (should be `items` or not wrapped in code tag)

### ISSUE THREAD

**Issue by trwnh** [w3c-social/activitypump#481]
> This Activity is added by the receiver as an `item` in the [inbox](https://www.w3.org/TR/activitypub/#inbox) OrderedCollection

there is no such `item` property, it should be `items` or otherwise should not be wrapped in code tags which suggests it is a property name.

### proposed solution

errata to either:

- s/item/items
- remove the `<code>` tags wrapping "item"

**Comment by TallTed** [comment]
Well, it shouldn't become "as an `items` in".

So perhaps "This Activity is added by the receiver to the `items` in the [inbox](https://www.w3.org/TR/activitypub/#inbox) OrderedCollection".

**Comment by trwnh** [comment]
on further reflection it should be removing the code tag because the
property may be orderedItems in most cases

**Comment by evanp** [comment]
I agree, it makes sense to rephrase this so that the `item` string does not suggest a property name. Let's add an erratum to that effect. Maybe something like:

> This Activity is added by the receiver to the [inbox](https://www.w3.org/TR/activitypub/#inbox) OrderedCollection

**Comment by evanp** [comment]
Removing WFC; will come back and work on an erratum.

**Comment by evanp** [comment]
I added an erratum as follows:

> Section 7.1 "Delivery" should read in part This Activity is added by the receiver to the inbox OrderedCollection.

**Comment by evanp** [comment]
Another erratum without a description of the problem; I'm re-tagging this as needs errata and we'll add some better text.

**Comment by trwnh** [comment]
### Error description

Section 7.1 "Delivery" reads in part, "This Activity is added by the receiver as an <code>item</code> in the [inbox](https://www.w3.org/TR/activitypub/#inbox) OrderedCollection". The use of the HTML code element suggests that there is an "item" property, but there isn't. (The actual property is "items" or "orderedItems".)

### Candidate correction 1

Remove the code formatting.
